src/ILVerify/src/Program.cs

```diff
@@ -3,42 +3,28 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
-using System.IO;
 using System.Collections.Generic;
 using System.CommandLine;
-using System.Reflection;
+using System.IO;
 using System.Linq;
-using System.Reflection.Metadata;
-using System.Reflection.Metadata.Ecma335;
+using System.Reflection;
 using System.Reflection.PortableExecutable;
-using System.Text;
-
-using Internal.TypeSystem;
-using Internal.TypeSystem.Ecma;
-using Internal.IL;
-
-using Internal.CommandLine;
 using System.Text.RegularExpressions;
-using System.Globalization;
-using System.Resources;
+using Internal.CommandLine;
 
 namespace ILVerify
 {
-    class Program
+    class Program : IResolver
     {
-        private const string DefaultSystemModuleName = "mscorlib";
         private bool _help;
 
-        private string _systemModule = DefaultSystemModuleName;
-        private Dictionary<string, string> _inputFilePaths = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);
-        private Dictionary<string, string> _referenceFilePaths = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);
+        private AssemblyName _systemModule;
```

Comment:
Shouldn't this be set to `AssemblyName("mscorlib")` by default?

Reply:
We don't know if the caller is using simple names or full names as identifiers.

Reply:
Oops. You were right :-)